aggregator/src/bin/aggregator.rs

```diff
@@ -250,6 +279,24 @@ struct Config {
     batch_aggregation_shard_count: u64,
 }
 
+#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
+struct GarbageCollectorConfig {
+    /// How frequently garbage collection is run, in seconds.
+    gc_frequency_s: u64,
+
+    /// The limit to the number of client report artifacts deleted for a single task by a single run
+    /// of the garbage collector.
+    report_limit: u64,
+
+    /// The limit to the number of aggregation jobs, and related aggregation artifacts, deleted for
+    /// a single task by a single run of the garbage collector.
+    aggregation_limit: u64,
+
+    /// The limit to the number of batches, and related collection artifacts, deleted for a single
+    /// task by a single run of the garbage collector.
+    collection_limit: u64,
```

Comment:
What I find most helpful with config flags (besides _what_ they do), is _why, or what factors should I consider before changing them. If that's open ended for now, that's fine, just wondering if you had an explicit thought in mind for how these should be derived.

Reply:
It's pretty open-ended, but: "large enough to keep up with the rate of creation, and not much larger than that." If we go below rate of creation we'll fall behind (of course), but at the same time, going too large will create larger-than-necessary transactions.